app/server/app/routes/data.js

```diff
@@ -9,6 +9,62 @@ const StreamingService = require("../utilities/streamingService");
 
 // mapping to dynamically build GET/POST endpoints and queries
 const mapping = {
+  assessmentUnits: {
+    tableName: "assessment_units",
+    idColumn: "id",
+    columns: [
+      { name: "id", alias: "id" },
+      { name: "assessmentunitid", alias: "assessmentUnitId" },
+      { name: "assessmentunitname", alias: "assessmentUnitName" },
+      { name: "assessmentunitstate", alias: "assessmentUnitState" },
```

Comment:
It looks like the name of this column has changed since the first schemas @pauldzy sent: it was initially labelled `assessmentunitstatus`. Will need to update this on the client side. Also, will the field also be changed in the `assessments` profile?

Reply:
@pauldzy Is this a mistake on the materialized views side?

![image](https://user-images.githubusercontent.com/46329268/206499013-d949ddf4-4b66-42e9-bc34-510f9bee5962.png)